cmd/crio/main.go

```diff
@@ -277,11 +277,11 @@ func main() {
 		},
 		cli.StringFlag{
 			Name:  "conmon",
-			Usage: "path to the conmon executable",
+			Usage: "path to the conmon executable (default: \"/usr/local/libexec/crio/conmon\")",
```

Comment:
I would suggest to use the value from the `defConf` variable like this:
```suggestion
			Usage: fmt.Sprintf("path to the conmon executable (default: %q)", defConf.Conmon),
```

Reply:
Or why not do
```golang
cli.StringFlag{
	Name:  "conmon",
        Value: defConf.Conmon,
	Usage: "path to the conmon executable",
},
```
and let `"github.com/urfave/cli"` add the default value text?

Reply:
Yes that is better, we just have to make sure that the current config vs cli overwrite behavior stays the same. :+1: